decidim-conferences/app/views/decidim/conferences/conference_program/show.html.erb

```diff
@@ -10,45 +10,31 @@ if respond_to?(:meeting_component) && meeting_component
 end
 %>
 <% add_decidim_meta_tags(title: t("conference_program.index.title", scope: "decidim")) %>
-<div class="section row">
-  <div class="columns mediumlarge-12">
-    <h3 class="section-heading"><%= t(".program") %></h3>
-    <div class="conference-programme">
-      <div class="row collapse programme-selector">
-        <div class="column medium-2">
-          <%= t(".day") %>
-        </div>
-        <div class="column medium-10">
-          <% if meeting_days.any? %>
-            <ul class="tabs" id="conference-day-tabs"
-              data-deep-link="true"
-              data-update-history="true"
-              data-deep-link-smudge="true"
-              data-deep-link-smudge-delay="500"
-              data-tabs>
-              <% meeting_days.each_with_index do |day, i| %>
-                <li class="tabs-title <%= "is-active" if i == 0 %>">
-                  <!-- Wrap the anchor inside a div to keep correct button styling -->
-                  <div>
-                    <a href="#conference-day-tab-<%= i %>" class="button button--nomargin button--sc hollow">
-                      <%= l day, format: :decidim_with_day_and_month_name %>
-                    </a>
-                  </div>
-                </li>
-              <% end %>
-            </ul>
-          <% end %>
-        </div>
-      </div>
-      <div class="tabs-content" data-tabs-content="conference-day-tabs">
-        <% meeting_days.each_with_index do |day, i| %>
-          <div class="programme-day tabs-panel<%= " is-active" if i == 0 %>" id="conference-day-tab-<%= i %>">
-            <% meetings_for_day(meeting_component, day, current_user).each do |meeting_days, meetings| %>
-              <%= render partial: "program_item", locals: { start_time: meeting_days[:start_time], end_time: meeting_days[:end_time], meetings: meetings.pluck(:meeting) } %>
-            <% end %>
-          </div>
-        <% end %>
-      </div>
+
+<%# NOTE: this page does not use a regular layout %>
```

Comment:
If you check the accessibility toolbar, you will find few accessibility issues to fix on this page:
![Program accessibility](https://user-images.githubusercontent.com/864340/197810366-8aa6572a-1e5b-4a8e-955f-39292fa359d8.png)